### ISSUE TITLE
Changing addButtonTitle and addMoreButtonTitle has no effect

### ISSUE THREAD

**Issue by philitell** [shubham-gupta-16/multi_image_picker_view#24]
Changing addButtonTitle and addMoreButtonTitle has no effect.

My configuration: 
```
MultiImagePickerView(
                  controller: _multiImagePickerController,
                  draggable: true,
                  showAddMoreButton: true,
                  showInitialContainer: true,
                  addButtonTitle: "test",
                  addMoreButtonTitle: "test2",
                  gridDelegate: const SliverGridDelegateWithFixedCrossAxisCount(
                    crossAxisCount: 2,
                    childAspectRatio: 1,
                  ),
                  onChange: (images) {
                    // callback to update images
                  },
                ),
```

The reason is that these parameters are not used in your source code.
see file: lib/src/multi_image_picker_view.dart - Line 100
Here "ADD IMAGES" is hardcoded. Please use addButtonTitle if it is set instead.
Same situation for "Add More" in Line 73 for addMoreButtonTitle

**Comment by nadialvy** [comment]
using the same issue

**Comment by nadialvy** [comment]
facing the same issue

**Comment by nadialvy** [comment]
https://github.com/shubham-gupta-16/multi_image_picker_view/pull/26 its fixed now, my PR has been merged!

**Comment by shubham-gupta-16** [comment]
Fixed in version `0.0.16`
